config/_default/languages.yaml

```diff
@@ -1,9 +1,11 @@
 en:
   languageName: English
   weight: 1
+  ### copyright (markdown can be used) ###
+  copyright: "Copyright &copy; {{< current_year >}} The Linux Foundation®. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our [Trademark Usage](https://www.linuxfoundation.org/trademark-usage) page. Linux is a registered trademark of Linus Torvalds. [Privacy Policy](https://www.linuxfoundation.org/privacy) and [Terms of Use](https://www.linuxfoundation.org/terms)."
+  title: "TODO Group // Talk openly, develop openly!!"
```

Comment:
Are the exclamation marks intentional?

Reply:
Ah, no. They were there for my tests. I have removed them now.